sycl/include/syclcompat/memory.hpp

```diff
@@ -448,6 +487,48 @@ static sycl::event combine_events(std::vector<sycl::event> &events,
 
 } // namespace detail
 
+namespace experimental {
+namespace detail {
+static inline std::vector<sycl::event>
+memcpy(sycl::queue q, const experimental::memcpy_parameter &param) {
+  auto to = param.to.pitched;
+  auto from = param.from.pitched;
+#ifdef SYCL_EXT_ONEAPI_BINDLESS_IMAGES
+  if (param.to.image_bindless != nullptr &&
+      param.from.image_bindless != nullptr) {
```

Comment:
@ziranzha thanks for your suggestion. I think its good. We also need `runtime_error` for now inside every `if, else-if, else-if` of the `SYCL_EXT_ONEAPI_BINDLESS_IMAGES` code path. Agreed?

Reply:
Agree